CandidateReranking.md

```diff
@@ -0,0 +1,368 @@
+
+#### Filter
+| Param    | Type | Description
+|----------|------|--------|
+| id | String | **Optional**: Unique identfier for the **Filter**.
+| name | String | **Optional**: Name of the **Filter**.
+| type | String | **Optional**: Type of **Filter** applied. Accepted values are *facet* or *range*. Service defaults to *facet*.
+| values | String[] | **Optional**: Values chosen to filter on from the original search request **Filter**.
+
+### Profile
+----------
+The profile is built with data from either **MY_SUPPLY** or **EDGE** (identified in the `source` param). Using data from each profile the service builds a RerankRequest Document which is sent along with features extracted from the query to the **Reranker**. The **Reranker** then returns the list of profiles in a new ordering which should be of more relevance to the query.
+
+**IMPORTANT: Though many fields on the profile are optional its encouraged to fill out the profile as fully as possible in order to get the best results from the Reranker**.
+
+#### Profile
+| Param    | Type | Description
+|----------|------|--------|
+| document_id | String | **Optional**: Unique CB identifier string for the candidate.
+| recent_jobs | [RecentJobs[]](#recentjob) | **Optional**: Recent jobs held by the candidate.
+| skills | [Skill[]](#skill) | **Optional**: Array of **Skill** objects. Skills associated with the candidate.
+| years_of_experience | Double | **Optional**: Total years of experience the candidate has as a double.
+| normalized_education_level | String[] | **Optional**: Normalized Educations associated with the candidate i.e., Master's Degree, Bachelor's Degree etc..
+| unnormalized_education_level | String[] | **Optional**: Unnormalized Educations associated with the candidate. i.e., masters degree, Master of Science, Masters etc..
+| major | String | **Optional**: The major or field of study the candidate's degree pertains to.
+| city | String | **Optional**: City that the candidate is located in.
+| state | String | **Optional**: State that the candidate lives in.
+| country | String | **Optional**: Country of the candidate.
+| latitude | Double | **Optional**: Location of **Latitude** associated with the canddiates location.
+| longitude | Double | **Optional**: Location of **Longitude** associated with the candidates location.
+
+#### RecentJob
+
+| Param    | Type | Description
+|----------|------|--------|
+| company_name | String | **Optional**: Company name where job was held for the candidate.
+| onet | Onet | **Optional**: ONet title associated with job.
+| carotene | Carotene | **Optional**: Carotene title associated with the job.
+| unclassified | Unclassified | **Optional**: Unclassified job title if the job does not have an ONet or Carotene title.
```

Comment:
Is this only returned if the `onet` and `carotene` are not? It sounds like that is the case, but maybe we could make that explicit?